main.py

```diff
@@ -286,28 +288,41 @@ def program_exit(status: int):  # so we don't need to import the entire sys modu
                                     m_set = ()
                                     for m in stats_data[player["Subject"]]:
                                         if m["match_id"] != coregame.match_id and m["match_id"] not in m_set:
+                                            if m["epoch"] > last_seen:
+                                                # get the last match they played with or against you
+                                                last_seen = m["epoch"]
+                                                # saves if they were on your team or not, defaults to true for old data
```

Comment:
When is `m[epoc]` going to be < 0?

Reply:
It is never going to be "< 0", it was just a way to ensure to get the last game played, since I wrongly assumed, that the games are stored in a dict, not an array - it should work without the if statement